src/yellow/components/button/_index.scss

```diff
@@ -130,24 +165,45 @@
       border-top: 3px solid transparent;
       box-sizing: border-box;
       content: '';
-      height: 20px;
+      height: 18px;
       left: 50%;
+      margin: -9px 0 0 -9px;
       position: absolute;
       top: 50%;
-      transform-origin: 0 0;
-      width: 20px;
+      width: 18px;
+
+      @keyframes spin {
+        100% {
+          transform: rotate(360deg);
+        }
+      }
     }
-  }
 
-  &--full-card {
-    border-bottom-left-radius: $size-border-radius-m;
-    border-bottom-right-radius: $size-border-radius-m;
-    border-top: get-border(size-border-width-none);
-    border-top-left-radius: 0;
-    border-top-right-radius: 0;
-  }
+    &[class~='btn--inverse'],
+    &[class~='btn--secondary'] {
+      &::before {
+        border-bottom: 3px solid $color-interface-content-dark;
+        border-left: 3px solid $color-interface-content-dark;
+        border-right: 3px solid $color-interface-content-dark;
```

Comment:
Instead of repeating the width and the style of the border, we could use`border-color` property to change only the color. That way we can change the border size once in the `btn--loading` class and reflect in this modifier.

Reply:
It makes sense, I ended up using the form that was already in the old component. I'm going to change!